### ISSUE TITLE
About technology statistics on the web

### ISSUE THREAD

**Issue by ROBERT-MCDOWELL** [open-source-flash/open-source-flash#90]
I would like to point out of how google & co. force people to use their technology  "standard", like chrome, webrtc and so on. The whole developer community just don't realize how google controls the internet standards and how they proceed to force developers and users to use what they wish.

Statistics are one of the first propaganda tool, fake or real, no one can check if what it shows on your screen is indeed true or false. But as evil is in the details, you will clearly see (if you make a little research by yourself) that what it's published is often not really true, but just a manner to project their wish to people mind.

Example, the repository open-source-flash of pakastin got more than 6777 stars and 132 watches in less than 2 months, webrtc (https://github.com/webrtc/webrtc-org) has painfully  41 watchers and 60 stars since years.

This show clearly that the protagonists are desperately trying to bring the whole worldwide web developer community to their side, and for sure it's easily understandable since a "standard", whatever it is, has no power but with a good amount of developers.

In conclusion, the best way to try to understand what's happening with all these technologies, standards and languages on the web we constantly need to ask questions and proof about all articles published, starting from its source.

**Comment by ROBERT-MCDOWELL** [comment]
PS: a nice article about JavaScript
https://www.leaseweb.com/labs/2013/07/10-very-good-reasons-to-stop-using-javascript/

**Comment by ROBERT-MCDOWELL** [comment]
derhuerst: well, I don't know why you reacted negatively to my article since I just report the state of what it is today....

**Comment by Shnoulle** [comment]
> Example, the repository open-source-flash of pakastin got more than 6777 stars and 132 watches in less than 2 months, webrtc (https://github.com/webrtc/webrtc-org) has painfully 41 watchers and 60 stars since years.

This example is bad : `github.com/webrtc/webrtc-org` is for website , who care of a website. `github.com/webrtc/samples` show different number.

**Comment by pakastin** [comment]
> PS: a nice article about JavaScript
> https://www.leaseweb.com/labs/2013/07/10-very-good-reasons-to-stop-using-javascript/

I'm nowadays a fullstack JavaScript developer (former Flash Developer), so please – there's nothing nice about that article..
